test/pytest/test_metrics.py

```diff
@@ -170,15 +125,21 @@ def test_log_location_var_snapshot_enabled():
     requests.post('http://127.0.0.1:8081/models?url=densenet161.mar')
     # We stop torchserve again here so that we can remove the LOG_LOCATION setting from environment variable
     test_utils.stop_torchserve()
+    print('Waiting to stop')
+    time.sleep(15)
     del os.environ['LOG_LOCATION']
+
     # In case of snapshot enabled, we get these three config files additionally in the custom directory
-    assert len(glob.glob(test_utils.ROOT_DIR + 'config/*startup.cfg')) >= 1
-    assert len(glob.glob(test_utils.ROOT_DIR + 'config/*shutdown.cfg')) >= 1
-    assert len(glob.glob(test_utils.ROOT_DIR + 'config/*snap*.cfg')) >= 1
-    for f in glob.glob(test_utils.ROOT_DIR + "*.log"):
+    assert len(glob.glob(path.join(test_utils.ROOT_DIR,'config/*startup.cfg'))) >= 1
+    if platform.system() != "Windows":
```

Comment:
Is snapshot not supported for windows native?

Reply:
There is an issue with Java shutdown hook on windows due to which the shutdown snapshot file is not generated hence this check. See the docs/snapshot.md for a specific notes in this PR.